src/clients/api/mutations/useApproveToken/getSpenderAddress.ts

```diff
@@ -0,0 +1,16 @@
+import { TokenId } from 'types';
```

Comment:
nit: Scattering utility functions like this doesn't seem ideal, regardless of whether they are only used once I think we should organize them in the `utilities` dir. It keeps everything organized in one place, you know where to look if you need a utility function, it decreases the possibility that it gets rewritten. It also means `useApproveToken` doesn't need to get moved to a directory

Reply:
I don't think that's the right approach here. The `utilities` folder should contain reusable utils, that's not the case here because it is specific to `useApproveToken`. It is the same thing as having a component that's only rendered in a specific page: we'd include it in the folder of that page only rather than making it a reusable component.

Reply:
Actually, you're right. I didn't realize that although this util is used only once, it is shared by two hooks: `useGetAllowance` and `useApproveToken`, so it should be a reusable hook. I'll make the change